CHANGELOG.rst

```diff
@@ -2,19 +2,35 @@
 Changelog
 =========
 
+`2023.7 <https://github.com/python/python-docs-theme/releases/tag/v2023.7>`_
+----------------------------------------------------------------------------
+
+- Do not append asset hashes on Sphinx 7.1+ (#137)
```

Comment:
This suggests a potential regression, that P-D-T no longer does the hashes -- whereas the functionality was added to Sphinx core instead.
```suggestion
- Fix compatability with Sphinx 7.1 (#137)
```